tests/test_app_e2e.py

```diff
@@ -0,0 +1,36 @@
+import pytest
+import os
+from threading import Thread
+from selenium import webdriver
+
+import trello_items
+import app
+
+@pytest.fixture(scope='module')
+def test_app():
+    # Create the new board & update the board id environment variable
+    board_id = trello_items.create_trello_board("TEST BOARD")
+    os.environ['TRELLO_BOARD_ID'] = board_id
+
+    # construct the new application
+    application = app.create_app()
+
+    # start the app in its own thread.
+    thread = Thread(target=lambda: application.run(use_reloader=False))
+    thread.daemon = True
+    thread.start()
+    yield application
+
+    # Tear Down
+    thread.join(1)
+    trello_items.delete_trello_board(board_id)
+
+@pytest.fixture(scope='module')
+def driver():
+    # local path to your webdriver download, please update accordingly
+    with webdriver.Chrome('C:\Work\DevOps-Course-Starter\drivers\chromedriver.exe') as driver:
```

Comment:
You can actually remove this path - `webdriver.Chrome()` (and update the README accordingly), then it works if it is on the user's PATH or in the root of the project.

Alternatively if you want it in the drivers folder for whatever reason, you can still make this a relative path - `webdriver.Chrome('drivers/chromedriver.exe')`